doc/develop/test/ztest.rst

```diff
@@ -111,6 +111,10 @@ This is achieved via fixtures in the following way:
    	zassert_equal(256, fixture->max_size);
    }
 
+Using memory allocated by a test fixture in a userspace thread, such as during execution of
+`ZTEST_USER` or `ZTEST_USER_F`, requires that memory to be declared userspace accessible. This is
```

Comment:
NIT: Could you prefix the `ZTEST_*` references with `:c:macro:` to match above references?  See under _Creating a test suite_

Reply:
Good point. Done and added for `ZTEST_DMEM` and `ZTEST_BMEM`.